app/controllers/api/v1/organizations_controller.rb

```diff
@@ -3,6 +3,14 @@
 module Api
   module V1
     class OrganizationsController < ApplicationController
+      before_action :authorization
+
+      # GET /organization_id/public
+
+      def public
+        @organization = Organization.find(params[:id])
```

Comment:
Is recommended to create a private method because is probably you need to reuse this line of code.